Example/Emission/EigenLikeNavigationController.m

```diff
@@ -14,18 +14,18 @@ - (void)viewDidLoad
   self.navigationBarHidden = YES;
   self.delegate = self;
 
-  UIButton *backButton = [self createBackButton];
-  [self.view addSubview:backButton];
-  [backButton constrainTopSpaceToView:self.topLayoutGuide predicate:@"12"];
-  [backButton alignLeadingEdgeWithView:self.view predicate:@"12"];
-  [backButton constrainWidth:@"40" height:@"40"];
-  _backButton = backButton;
+//  UIButton *backButton = [self createBackButton];
```

Comment:
You had to do this for home too, it could be worth adding the feature to have this hide on specific view controllers?